ci-bootstrap/terraform/terraform.tfstate

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Don't check in tfstate files as they can contain credentials - should always add them to your `.gitignore` file

Reply:
Hm, not sure why I added it to the commit - probably by mistake. Good point. Added to gitignore.